migrations/migration-9/pin-set.js

```diff
@@ -86,15 +82,15 @@ function hash (seed, key) {
   const buffer = new Uint8Array(4)
   const dataView = new DataView(buffer.buffer)
   dataView.setUint32(0, seed, true)
-  const encodedKey = uint8ArrayFromString(toB58String(key))
+  const encodedKey = uint8ArrayFromString(key.toString())
```

Comment:
⚠️  Is this intentional change, seems like keys for v1 CID will get encoded differently now.